src/global-firestore-data/onboarding-data.ts

```diff
@@ -15,30 +15,35 @@ export const setAppOnboardingData = (
     majors: onboarding.major.map(acronym => ({ acronym })),
     minors: onboarding.minor.map(acronym => ({ acronym })),
     gradPrograms: onboarding.grad ? [{ acronym: onboarding.grad }] : [],
-    exam: onboarding.exam,
+    // TODO @bshen migration script from type to examType
+    exam: onboarding.exam.map(e => ({
+      ...e,
+      type: e.examType || e.type,
+      examType: e.type || e.examType,
+    })),
```

Comment:
I strongly think it's worth the migration script, since `type` is essentially meaningless (or it has too many meanings). I'll make sure it gets run this summer or next semester :')